bundles/org.openhab.binding.surepetcare/src/test/java/org/openhab/binding/surepetcare/internal/data/SurePetcarePetLocationTest.java

```diff
@@ -0,0 +1,48 @@
+package org.openhab.binding.surepetcare.internal.data;
+
+import static org.junit.Assert.assertEquals;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.junit.Test;
+import org.openhab.binding.surepetcare.internal.GsonColonDateTypeAdapter;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class SurePetcarePetLocationTest {
+
+    private static final Gson gson = new GsonBuilder()
```

Comment:
You should use the actual Gson used in the binding. Otherwise you might end up with a test that succeeds, but fails because the actual gson initialized in the binding is different. Possible solution put a static instance in a constants class that is used everywhere.